mendel/core.py

```
@@ -685,6 +692,27 @@ def _track_event(self, event):
         if r.code != 200:
             print red('Unable to track deployment event in graphite (HTTP %s)' % r.code)
 
+    def _track_event_api(self, event):
+        """
+        Track who deployed what service and what the release dir is to an external REST API
+        """
+        if not self._track_event_endpoint:
+            print red('Unable to track deployment event in custom api, no api endpoint configured in ~/.mendel.conf')
+            return
+
+        data = {
+            'service': self._api_service_name or self._service_name,
+            'host': env.host_string,
+            'deployer': getpass.getuser(),
+            'event': event
+        }
+
+        url = 'http://%s' % self._track_event_endpoint
+
+        r = requests.post(url, data)
```

it's possible this throws an exception, probably want to catch and swallow w/ a warning that tracking the event failed. I should have done that above as well. Also, why introducing python `requests` library? just adding additional dependencies to mendel, can use urllib2 as above